runtime/runtime/src/lib.rs

```diff
@@ -661,7 +665,7 @@ impl Runtime {
                 validator_reward -= receiver_reward;
                 account.amount = safe_add_balance(account.amount, receiver_reward)?;
                 set_account(state_update, account_id, account);
-                state_update.commit();
+                state_update.commit(StateChangeCause::ValidatorAccountsUpdate);
```

Comment:
This is for contract execution kickback to the contract developer. It's a 30% kickback for burnt gas. So it's not validator accounts update.

Probably fine to use `ReceiptProcessing` here as well for the same receipt.